easyrsa3/easyrsa

```diff
@@ -941,34 +949,47 @@ Matching file found at: "
 		die "Failed to sign '$name'"
 	}
 
-	# inline it
-	if [ $EASYRSA_INLINE ]; then
-		inline_creds
-	fi
 } # => build_full()
 
-# Create inline credentials file for this node
-inline_creds ()
+# Create inline credentials file
+build_inline ()
 {
-	[ -f "$EASYRSA_PKI/$EASYRSA_REQ_CN.creds" ] \
-	&& die "Inline file exists: $EASYRSA_PKI/$EASYRSA_REQ_CN.creds"
+	# TODO: Improve build_inline integration. eg: renew/revoke and ta.key
+	verify_ca_init
+	name="$1"
+	ca_file="$EASYRSA_PKI/ca.crt"
+	crt_file="$EASYRSA_PKI/issued/$name.crt"
+	key_file="$EASYRSA_PKI/private/$name.key"
+	out_file="$EASYRSA_PKI/inline/$name.conf"
```

Comment:
Again, ovpn instead of conf.

Reply:
GUI  breakage

Reply:
GUI breakage is not right. OpenVPN Access Server and Private Tunnel has always used `.ovpn` since the very beginning.  This extension is also tied to the clients being installed.  The `.conf` is more commonly used (but not consistently; see comment below) on *nix platforms.


There are no consistent configuration file naming on *nix. Config files may not use any extension at all, `.cf`, `.cnf`, `.cfg`.  On top of that you have all the variants indicating formatting style, like `.ini`, `.xml`, `.json` etc, etc.  Some even use `.txt`.  The directory the file is found in is more commonly an indicator it is a config file; like using `/etc`, `/usr/local/etc`, `~/.local`, `~/.config`, etc, etc.  More details in the FHS spec: https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.pdf

Reply:
Nice to see you read the entire PR ...

Reply:
@dsommers I love your _English_
But .. try not to not not people, especially if not _notting_ someone is not something you are good at.

Reply:
The point is: GUI breakage is **exactly** the problem `...`